### PR TITLE
Write up - Interview with Francis

### LINKED ISSUE #20: Interview Francis Irving about first years of site
## Q: What factors caused WDTK to grow?

Unmeasurable, but very insights will be valuable

### PR DESCRIPTION
**Pull request by Gemmamysoc**
Have written up summary and changed README

Closes #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,5 +190,15 @@ A collection of experiments and research to learn about what makes an Alaveteli
         <strong>No.</strong> Not enough traffic to make an impact.
       </td>
     </tr>
+ <tr>
+      <td>
+        <a href="experiments/what-factors-caused-WDTK-to-grow.md">
```
Comment (lizconlan) [inline README.md]: house style - filenames should be lower case (file should be renamed as well but I can't work out how to comment that \*sigh\*)

File: experiments/what-factors-caused-WDTK-to-grow.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+## Answer
+
+Summary of most important things:
+
+
+
+- Getting a **Google grant to do Google Adwords**. Advice: Make sure you appear on the search page of when people are searching for FOI especially. Before Google Adwords WDTK did not appear at all on search results, then after they got the grant they were listed above government adverts/sites. 
+
+
+- Tom Steinberg had lots of **contacts in the media** so got the site featured in mainstream media
```
Comment (lizconlan) [inline experiments/what-factors-caused-WDTK-to-grow.md]: ignorable suggestion: "media contacts" for "contacts in the media"?

Comment (lizconlan) [inline experiments/what-factors-caused-WDTK-to-grow.md]: oh, and a full stop at the end 😃 (so hard to know whether to or not when dealing with bullet points!)

File: experiments/what-factors-caused-WDTK-to-grow.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+
+- mySociety had built up a **strong community of people interested in civic tech and FOI** so there was a **desire for a site like WDTK well before it was built**. This community was built up over a number of years, through newspapers, blogs, mailing lists. mySociety also did a Call for Proposals for civic tech projects when charity first launched - based on an article Tom wrote about the concept of digital democracy etc. WDTK was suggested by Francis and someone else in the 2nd call. The best project ideas were taken and an event for 50-100 people was held to launch ideas. So community was already invested in the idea before it was even built.
```
Comment (lizconlan) [inline experiments/what-factors-caused-WDTK-to-grow.md]: suggestion: when _the_ charity first launched

File: experiments/what-factors-caused-WDTK-to-grow.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+- They didn’t send out newsletters or anything to users. **Users found like-minded people working on similar issues via the Comments on requests, and then directly messaged each other**.
+
+
+- They had a **really good team of people with a variety of skills/interests**. Tom was really good at the marketing/promotion of the site, and it took him a considerable amount of time. So making sure Alaveteli partners have someone dedicated to marketing is really important. Advice: Try and find a volunteer who would find marketing the site an interesting challenge. 
+
+
+- There wasn’t any social media back then really. So they **used mailing lists and blogs, and mainstream media**. Mainstream media was easier to exploit in those days - less of a corporate hold. 
+
+Here are the [full notes](https://docs.google.com/document/d/16sUvKLLi6RnFYCjh1UryXuZK-lic5O-4kRkqxdkBz6w/edit) from our chat.
```
Comment (lizconlan) [inline experiments/what-factors-caused-WDTK-to-grow.md]: Probably shouldn't be an edit link, have changed the Google doc permissions to "Anyone with the link can view" (it's in a mySoc folder tho so mySociety staff with access to the folder can still edit)

File: README.md (hunk 1 above)
Comment (lizconlan) [inline README.md]: not the end of the world tbh, just a dev being fussy!

File: experiments/what-factors-caused-WDTK-to-grow.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+- In UK **FOI was a new thing** at the time they launched - had more media buzz and interest. 
+
+
+- The **internet in general was pretty new**, so the thought of using it to request information and publish responses was very novel. 
+
+
+- They **approached lots of other FOI enthusiasts** - Campaign for FOI etc to tell them about site. Who are the equivalents to Ben Goldacre in other countries? 
+
+
+- Via Tom’s media contacts they got **various articles in the press**. Examples include: Highlighting authorities who used ridiculous excuses not to reply and had to be chased loads of times (ridiculing them basically and quoting individual FOI officers); silly requests got a lot of coverage (although could be seen as devaluing FOI)
```
Comment (lizconlan) [inline experiments/what-factors-caused-WDTK-to-grow.md]: full stop missing

File: experiments/what-factors-caused-WDTK-to-grow.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+
+- They didn’t send out newsletters or anything to users. **Users found like-minded people working on similar issues via the Comments on requests, and then directly messaged each other**.
+
+
+- They had a **really good team of people with a variety of skills/interests**. Tom was really good at the marketing/promotion of the site, and it took him a considerable amount of time. So making sure Alaveteli partners have someone dedicated to marketing is really important. Advice: Try and find a volunteer who would find marketing the site an interesting challenge. 
+
+
+- There wasn’t any social media back then really. So they **used mailing lists and blogs, and mainstream media**. Mainstream media was easier to exploit in those days - less of a corporate hold.
```
Comment (lizconlan) [inline experiments/what-factors-caused-WDTK-to-grow.md]: comma instead of full stop between "really" and "so they"?

**Comment by Gemmamysoc** [comment]
Have made the changes suggested. Please could you merge @lizconlan ? Thanks!

**Comment by lizconlan** [comment]
@Gemmamysoc technically the next step is re-review I think (but I already looked tbh!) so yeah, sure 😃

**Comment by garethrees** [comment]
@lizconlan probably worth squashing in to one commit 👍  thanks!

**Comment by lizconlan** [comment]
@garethrees will do, thanks for the nudge :medal_sports:

**Pull request merged**